lib/go/test/flow_idtable_staking_test.go

```diff
@@ -2664,6 +2664,67 @@ func TestIDTable(t *testing.T) {
 
 	})
 
+	t.Run("Should be able cancel unstake request for delegator", func(t *testing.T) {
+
```

Comment:
This carries on the previous state, so at this point, there is 40k delegated, right?

one thing that could make these test additions easier to review is to have comments indicating the start state for the `Run`

Reply:
yeah that is a good idea. I can start cleaning these up and adding those comments